<?xml version="1.0" encoding="UTF-8"?>
<testng-results skipped="0" failed="1" total="1" passed="0">
  <reporter-output>
  </reporter-output>
  <suite name="Default suite" duration-ms="6860" started-at="2020-08-17T13:54:42Z" finished-at="2020-08-17T13:54:49Z">
    <groups>
    </groups>
    <test name="Default test" duration-ms="6860" started-at="2020-08-17T13:54:42Z" finished-at="2020-08-17T13:54:49Z">
      <class name="Takeaway.com.Assignment.Case_001">
        <test-method status="PASS" signature="initialize()[pri:0, instance:Takeaway.com.Assignment.Case_001@3b0fe47a]" name="initialize" is-config="true" duration-ms="961" started-at="2020-08-17T15:54:42Z" finished-at="2020-08-17T15:54:43Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- initialize -->
        <test-method status="FAIL" signature="caseOne_CreateOrderAndPayWithClosestAmout()[pri:0, instance:Takeaway.com.Assignment.Case_001@3b0fe47a]" name="caseOne_CreateOrderAndPayWithClosestAmout" duration-ms="5777" started-at="2020-08-17T15:54:43Z" finished-at="2020-08-17T15:54:48Z">
          <exception class="java.lang.NullPointerException">
            <full-stacktrace>
              <![CDATA[java.lang.NullPointerException
	at resources.ReadJsonData.customerDetails(ReadJsonData.java:51)
	at Takeaway.com.Assignment.Case_001.caseOne_CreateOrderAndPayWithClosestAmout(Case_001.java:38)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:80)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:714)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:901)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1231)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:127)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:111)
	at org.testng.TestRunner.privateRun(TestRunner.java:767)
	at org.testng.TestRunner.run(TestRunner.java:617)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:334)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:329)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:291)
	at org.testng.SuiteRunner.run(SuiteRunner.java:240)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1198)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1123)
	at org.testng.TestNG.run(TestNG.java:1031)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.NullPointerException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- caseOne_CreateOrderAndPayWithClosestAmout -->
        <test-method status="PASS" signature="teardown()[pri:0, instance:Takeaway.com.Assignment.Case_001@3b0fe47a]" name="teardown" is-config="true" duration-ms="111" started-at="2020-08-17T15:54:48Z" finished-at="2020-08-17T15:54:49Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- teardown -->
      </class> <!-- Takeaway.com.Assignment.Case_001 -->
    </test> <!-- Default test -->
  </suite> <!-- Default suite -->
</testng-results>
